core-bundle/src/Resources/contao/library/Contao/Model/Registry.php

```diff
@@ -153,12 +165,12 @@ public function register(Model $objModel)
 
 		$strTable = $objModel->getTable();
 
-		if (!\is_array($this->arrAliases[$strTable]))
+		if (!\is_array($this->arrAliases[$strTable] ?? null))
```

Comment:
```suggestion
		if (!isset($this->arrAliases[$strTable]))
```

We probably do not need the `is_array()` check at all, because `$this->arrAliases[$strTable]` is either an array or not set at all. WDYT?

Reply:
That's what I thought as well.